plugins/modules/win_net_adapter_feature.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, ライトウェルの人 <jiro.higuchi@shi-g.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# this is a windows documentation stub.  actual code lives in the .ps1
+# file of the same name
+
+DOCUMENTATION = r'''
+---
+module: win_net_adapter_feature
+short_description: Enable or disable certain network adapters.
+description:
+     - Enable or disable ms_tcpip6 of a certain network adapter or all the network adapters.
+options:
+  interface:
+    description:
+      - Name of Network Adapter Interface. For example, 'Ethernet0' or '*'.
```

Comment:
When quoting values, best to use `C()` instead of `''`. Using `C()` will have our doc generator emphasis the text and it's designed to be used for module option values.

```suggestion
      - Name of Network Adapter Interface. For example, C(Ethernet0) or C(*).
```

Another common formatter to use is `I()` which is designed to be used with module options themselves, like `I(state)` and so on.